web/tests/functional/statistics/test_statistics.py

```diff
@@ -48,6 +49,12 @@ def setUp(self):
         print("'analyze' reported statistics collector-compatibility? " +
               str(self.stats_capable))
 
+        try:
+            self.force_stats_capable = \
+                bool(util.strtobool(os.environ['CC_TEST_FORCE_STATS_CAPABLE']))
+        except ValueError:
+            self.force_stats_capable = False
```

Comment:
Wouldn't it be better to use logic like this:

```suggestion
        if not self.stats_capable:
            try:
                self.stats_capable = \
                    bool(util.strtobool(os.environ.get('CC_TEST_FORCE_STATS_CAPABLE', '0')))
            except ValueError:
                pass
```

This way there would be no need to write long conditional expression in every test cases (`if not self.force_stats_capable and not self.stats_capable:`).